.travis.yml

```diff
@@ -38,8 +38,10 @@ _android: &_android
 matrix:
   include:
     # one local test, without saucelabs
-    - env: PLATFORM=local
+    - env: PLATFORM=local/ios-10.0
       <<: *_ios
+    - env: PLATFORM=local/android-5.1
```

Comment:
Specific reason you chose 5.1?

Per browser market share, 8.1 is the most popular one right now already: http://gs.statcounter.com/os-version-market-share/android/mobile-tablet/worldwide Does this make sense to use that here?